model-optimizer/mo/front/tf/extractor.py

```diff
@@ -14,11 +14,7 @@
  limitations under the License.
 """
 
-from mo.front.tf.extractors.concat import tf_concat_ext
-from mo.front.tf.extractors.fused_bn import tf_fused_bn_extractor
 from mo.front.tf.extractors.native_tf import native_tf_node_extractor
-from mo.front.tf.extractors.pack import tf_pack_ext
-from mo.front.tf.extractors.random_uniform import tf_random_uniform_ext
 from mo.front.tf.extractors.utils import get_tf_node_port
```

Comment:
The `get_tf_node_port` function is not used in this file

Reply:
src_node, src_port = get_tf_node_port(src_node_id)   --  32 line